language-design.md

```diff
@@ -73,8 +101,10 @@ sympathize more with the tenor of the article or the tenor of the majority of
 the commenters?
 
 **Response**
+The essence of this conflict I think is logical vs non-logical syntax. Personally, I find some solace in programming because it is completely logical. The English language is not completely logical but I still enjoy using it for exactly that reason. English is a language all the same as any programming language. We could imagine that eventually, English will be a programming language and we will be able to speak it to computers and have them understand it. But technology is not that far along yet so as of now we have to speak to computers in a more logical language. This is solely because computers think with electrical circuits and bit (purely logical) and humans think with brain cells (which leaves room for interpretation). Some people would prefer computers think more like humans, but other technical minded people may always prefer to think like computers. -Daniel
 
```

Comment:
"We could imagine that eventually, English will be a programming language and we will be able to speak it to computers and have them understand it."

What about Siri?